builder/osc/common/step_source_omi_info_test.go

```diff
@@ -31,7 +31,7 @@ func getState() (multistep.StateBag, error) {
 
 func TestMostRecentOmiFilter(t *testing.T) {
 	stepSourceOMIInfo := StepSourceOMIInfo{
-		SourceOmi: "ami-7cab7c18",
+		SourceOmi: "ami-e58ac287",
```

Comment:
Use ami-68ed4301 for non obsolete OMI (if you really need a Rocky image => copy the OMI in our account)